user_tools/src/spark_rapids_tools/tools/qualx/preprocess.py

```diff
@@ -481,8 +487,8 @@ def combine_tables(table_name: str) -> pd.DataFrame:
         'Scan DeltaCDFRelation',
         'Scan ExistingRDD Delta Table Checkpoint',
         'Scan ExistingRDD Delta Table State',
-        'Scan JDBCRelation',
-        'Scan parquet ',  # trailing space is also in default sql op name
+        'Scan jdbc',
+        'Scan parquet',
```

Comment:
QQ: Do we need to add here all the remaining `Scan {format}`, and `Scan ExistingRDD`?
CC: @leewyang

Reply:
@leewyang - Could you please take a look at the changes in this file. I am not certain if we need to add the the remaining formats or if anything should be removed.

Reply:
This `dynamic_op_labels` list is only used to normalize exec/ops that have dynamic components to them, e.g. `Scan jdbc XXXX`.  If these dynamic components are being removed going forward, this code will basically become a no-op (and could be removed entirely).